components/brave_wallet/browser/swap_service.cc

```diff
@@ -166,78 +166,88 @@ brave_wallet::mojom::BraveSwapFeeResponsePtr GetBraveFeeInternal(
 }
 
 GURL Append0xSwapParams(const GURL& swap_url,
-                        const brave_wallet::mojom::SwapParams& params) {
+                        const brave_wallet::mojom::SwapQuoteParams& params) {
   GURL url = swap_url;
-  if (!params.taker_address.empty()) {
-    url = net::AppendQueryParameter(url, "takerAddress", params.taker_address);
+  if (!params.from_account_id->address.empty()) {
+    url = net::AppendQueryParameter(url, "takerAddress",
+                                    params.from_account_id->address);
   }
-  if (!params.sell_amount.empty()) {
-    url = net::AppendQueryParameter(url, "sellAmount", params.sell_amount);
+  if (!params.from_amount.empty()) {
+    url = net::AppendQueryParameter(url, "sellAmount", params.from_amount);
   }
-  if (!params.buy_amount.empty()) {
-    url = net::AppendQueryParameter(url, "buyAmount", params.buy_amount);
+  if (!params.to_amount.empty()) {
+    url = net::AppendQueryParameter(url, "buyAmount", params.to_amount);
   }
-  if (!params.buy_token.empty()) {
-    url = net::AppendQueryParameter(url, "buyToken", params.buy_token);
+  if (!params.to_token.empty()) {
+    url = net::AppendQueryParameter(url, "buyToken", params.to_token);
   }
-  if (!params.sell_token.empty()) {
-    url = net::AppendQueryParameter(url, "sellToken", params.sell_token);
+  if (!params.from_token.empty()) {
+    url = net::AppendQueryParameter(url, "sellToken", params.from_token);
   }
 
   auto brave_swap_fee_params = brave_wallet::mojom::BraveSwapFeeParams::New();
-  brave_swap_fee_params->chain_id = params.chain_id;
-  brave_swap_fee_params->taker = params.taker_address;
-  brave_swap_fee_params->input_token = params.sell_token;
-  brave_swap_fee_params->output_token = params.buy_token;
+  brave_swap_fee_params->chain_id = params.from_chain_id;
+  brave_swap_fee_params->taker = params.from_account_id->address;
+  brave_swap_fee_params->input_token = params.from_token;
+  brave_swap_fee_params->output_token = params.to_token;
   const auto& brave_fee = GetBraveFeeInternal(std::move(brave_swap_fee_params));
   if (brave_fee && brave_fee->has_brave_fee) {
     url = net::AppendQueryParameter(url, "buyTokenPercentageFee",
                                     brave_fee->fee_param);
   }
 
-  url = net::AppendQueryParameter(
-      url, "slippagePercentage",
-      base::StringPrintf("%.6f", params.slippage_percentage));
+  double slippage_percentage = 0.0;
+  if (base::StringToDouble(params.slippage_percentage, &slippage_percentage)) {
+    url = net::AppendQueryParameter(
+        url, "slippagePercentage",
+        base::StringPrintf("%.6f", slippage_percentage / 100));
+  }
 
```

Comment:
`slippagePercentage` now might not appear in url. Is it ok?(In terms of possible money loss)

Reply:
and same for `slippageBps` below

Reply:
In that case, the default value is assumed by 0x, which is 0.01 (or 1%).